### PR TITLE
Compatibility React 15.5.0

### PR DESCRIPTION
**Pull request by Anomen**
Small PR to remove the deprecated React.PropTypes for React 15.5.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,6 +51,7 @@
     "node-libs-browser": "^1.0.0",
     "node-sass": "^3.2.0",
     "postcss-loader": "^0.13.0",
+    "prop-types": "^15.5.8",
```
Comment (rafeememon) [inline package.json]: This should probably go in `dependencies` as per https://github.com/reactjs/prop-types/blob/master/README.md#how-to-depend-on-this-package

Comment (Anomen) [inline package.json]: :+1: Fixed

**Comment by reintroducing** [comment]
Any possibility of this getting merged? Appreciate it.

**Comment by souporserious** [comment]
Sorry, I've been super busy, I'll get this merged ASAP. Does someone want to take this project over? I've been focusing my efforts on [React Popper](https://github.com/souporserious/react-popper) recently and don't use this anymore.

**Comment by reintroducing** [comment]
Appreciate it and your work on this in general. Thank you.

**Pull request merged**